docs/user/faq.rst

```diff
@@ -610,33 +610,85 @@ method, making it compatible with ``boto3``\'s
    pattern can be applied to any storage API that supports streaming directly
    from a file-like object.
 
-How do I consume a query string that has a JSON value?
-------------------------------------------------------
-Falcon defaults to treating commas in a query string as literal characters
-delimiting a comma separated list. For example, given
-the query string ``?c=1,2,3``, Falcon defaults to adding this to your
-``request.params`` dictionary as ``{'c': ['1', '2', '3']}``. If you attempt
-to use JSON in the value of the query string, for example ``?c={'a':1,'b':2}``,
-the value will get added to your ``request.params`` in a way that you probably
-don't expect: ``{'c': ["{'a':1", "'b':2}"]}``.
+How do I retrieve a JSON value from the query string?
+-----------------------------------------------------
+To retrieve a JSON-encoded value from the query string, Falcon provides the
+:meth:`~falcon.Request.get_param_as_json` method, an example of  which is given
+below:
+
+.. code:: python
+
+    import falcon
+
+
+    class LocationResource:
+
+        def on_get(self, req, resp):
+            places = {
+                'Chandigarh, India': {
+                    'lat': 30.692781,
+                    'long': 76.740875
+                },
+
+                'Ontario, Canada': {
+                    'lat': 43.539814,
+                    'long': -80.246094
+                }
+            }
+
+            coordinates = req.get_param_as_json('place')
+
+            for (key, value) in places.items():
+                if coordinates == value:
+                    place = key
+
+            resp.media = {
+                'place': place
+            }
+
+
+    app = falcon.API()
+    app.add_route('/locations', LocationResource())
+
+In the example above, ``LocationResource`` expects a query string containing
+a JSON-encoded value named ``'place'``. This value can be fetched and
+decoded from JSON in a single step with the
+:meth:`~falcon.Request.get_param_as_json` method. Given a request URL
+such as:
+
+    ``/locations?place={'lat': 43.539814, 'long': -80.246094}``
+
+The `coordinates` variable will be set to a :class:`dict` as expected.
+
+By default, the :attr:`~falcon.RequestOptions.auto_parse_qs_csv` option is
+set to ``False``. The example above assumes this default.
+
+On the other hand, when :attr:`~falcon.RequestOptions.auto_parse_qs_csv` is set
+to ``True``, Falcon treats commas in a query string as literal characters
+delimiting a comma-separated list. For example, given the query string
+``?c=1,2,3``, Falcon defaults to adding this to your ``request.params``
+dictionary as ``{'c': ['1', '2', '3']}``. If you attempt to use JSON in the
+value of the query string, for example ``?c={'a':1,'b':2}``, the value will get
```

Comment:
Same here, single quotes make it invalid JSON (although it would be broken apart by the `auto_parse_qs_csv` option anyway according to the explanation).